content/modules/redisearch/redisearch-2-upgrade.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: Upgrading a Database to RediSearch 2.x
+description:
+weight: 30
+alwaysopen: false
+categories: ["Modules"]
+module: RediSearch
+---
+RediSearch 2.x includes some significant changes to the architecture of RediSearch that provide improved functionality.
```

Comment:
I believe this needs some prerequisites no?  Python?  Which pip dependencies need to be installed? @MeirShpilraien

Reply:
@K-Jo From my experience you just need to install python (that's python 2). I'll add it.

Reply:
@MeirShpilraien Can you confirm?

Reply:
No pip dependencies are needed, all the imports are standard.